frontend/packages/console-app/src/components/quick-starts/utils/quick-start-utils.ts

```diff
@@ -11,3 +12,43 @@ export const getQuickStartStatus = (
   quickStartID: string,
 ): QuickStartStatus =>
   (allQuickStartStates?.[quickStartID]?.status as QuickStartStatus) ?? QuickStartStatus.NOT_STARTED;
+
+export const getQuickStartStatusCount = (
+  allQuickStartStates: AllQuickStartStates,
+  quickStarts: QuickStart[],
+): Record<QuickStartStatus, number> => {
+  return quickStarts.reduce(
+    (totals, item) => {
+      totals[getQuickStartStatus(allQuickStartStates, item.metadata.name)]++;
+      return totals;
+    },
+    {
+      [QuickStartStatus.IN_PROGRESS]: 0,
+      [QuickStartStatus.COMPLETE]: 0,
+      [QuickStartStatus.NOT_STARTED]: 0,
+    },
+  );
+};
+
+export const filterQuickStarts = (
+  quickStarts: QuickStart[],
+  filterText: string,
+  statusFilters: string[],
+  allQuickStartStates: AllQuickStartStates,
+): QuickStart[] => {
+  const searchText = filterText.toLowerCase();
+  return quickStarts.filter(
+    ({ metadata: { name }, spec: { displayName, prerequisites = [], description } }) => {
+      const matchesFilter =
+        statusFilters.length > 0
+          ? statusFilters.includes(getQuickStartStatus(allQuickStartStates, name))
+          : true;
+      const matchesText =
+        fuzzy(searchText, displayName.toLowerCase()) ||
+        fuzzy(searchText, description.toLowerCase()) ||
+        prerequisites.some((text) => fuzzy(searchText, text.toLowerCase()));
```

Comment:
I think we should use `includes` for string matching as fuzzy search has strange results. In our catalogs we use `includes` and not fuzzy search.

fuzzy search matches so long as the next character in the search string is found between the previous character match and before the end of the string to match against.

![image](https://user-images.githubusercontent.com/14068621/100668567-1b06b700-332a-11eb-8a68-a13fab126561.png)

Reply:
My bad! Sorry @rottencandy ...

Reply:
Let's get UX input here.
cc @lwrigh  

@spadgett what's the history in console for when to use `fuzzy` vs string inculdes?

Reply:
I feel like fuzzy is fine when matching against short strings like a name. But it falls apart when you have a very long description.

Reply:
@christianvogt I agree with using` includes` instead of `fuzzy` search here. The example you showed is misleading and negatively impacts the UX so I would avoid scenarios like that. Most of the time I would imagine the user is searching by exact term so `includes `makes most sense to me.

Reply:
Updated to use `includes`